python/ray/_private/utils.py

```diff
@@ -1156,3 +1156,15 @@ def check_dashboard_dependencies_installed() -> bool:
         return True
     except ImportError:
         return False
+
+
+def lazy_import(name):
+    if name in sys.modules:
+        return
+    spec = importlib.util.find_spec(name)
```

Comment:
If the package doesn't exist (i.e. Pandas isn't installed), this will return `None`, which will cause the below `spec.loader` to fail with an attribute error. We should check to see if this is `None` here and raise the canonical `ModuleNotFoundError` if so.

Reply:
cc @ericl pretty sure this is the source of the test error